alpha/action/render.go

```diff
@@ -296,22 +296,33 @@ func populateDBRelatedImages(ctx context.Context, cfg *declcfg.DeclarativeConfig
 }
 
 func bundleToDeclcfg(bundle *registry.Bundle) (*declcfg.DeclarativeConfig, error) {
-	bundleProperties, err := registry.PropertiesFromBundle(bundle)
+	objs, props, err := registry.ObjectsAndPropertiesFromBundle(bundle)
```

Comment:
This is the breaking change that go-apidiff is complaining about. This function was introduced with FBC code, so it hasn't been in the wild long.

I'm perfectly willing to refactor to avoid a breaking change if others think its necessary here.